import/webview/plugin.cpp

```diff
@@ -74,6 +79,33 @@ void SailfishOSWebViewPlugin::initializeEngine(QQmlEngine *engine, const char *u
     webEngine->addObserver(QStringLiteral("clipboard:setdata"));
 }
 
+void SailfishOSWebViewPlugin::initUserAgentOverrides(const QString &path)
+{
+    QString mozillaDir = QString("%1/.mozilla/").arg(path);
+    QDir dir(mozillaDir);
+    if (!dir.exists())
+        dir.mkpath(dir.path());
+
+    QFile ua(mozillaDir + MOZILLA_DATA_UA_UPDATE);
+    if (!ua.exists()) {
```

Comment:
I may be misreading the logic here, but it looks like once the file is created, it will never be regenerated. Is this correct behaviour? It seems that if `ua-update.json.in` changes then this file should be regenerated too.

Reply:
New version which adds missing overrides from ua-update.json.in.